clients/docs/includes/configs/cloud/restproxy-sr.config

```diff
@@ -0,0 +1,16 @@
+# Kafka
```

Comment:
Also need a file for restproxy.config?

Reply:
I didn't create one because we do not have a use case for `restproxy.config` in the Cloud UI right now. Should I still add it regardless?

Reply:
Use case: a user wants to use RP without CCSR?  How is that handled in the console?